lib/tasks/lae.rake

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 require 'rsolr'
 require 'faraday'
-require 'webmock' unless Rails.env.production?
+require 'webmock' unless Rails.env.production? || Rails.env.staging?
```

Comment:
There's another similar check lower down in this file.

Reply:
oops; I only searched for webmock. that should be fixed now.